src/distilabel/pipeline/local.py

```diff
@@ -55,9 +55,9 @@
 _SUBPROCESS_EXCEPTION: Union[Exception, None] = None
 
 
-def _init_worker(queue: "Queue[Any]") -> None:
+def _init_worker(queue: "Queue[Any]", log_filename: str = "pipeline.log") -> None:
     signal.signal(signal.SIGINT, signal.SIG_IGN)
-    setup_logging(queue)
+    setup_logging(queue, filename=log_filename)
```

Comment:
I would just leave `setup_logging(queue)` because like this it seems that the child processes needs also to setup the log file, when they don't have to anything with the log file.

Reply:
totally!